taskcluster/scripts/bitrise-schedule.py

```diff
@@ -173,9 +175,11 @@ async def download_log(client, build_slug):
 
     response = await do_http_request_json(client, url)
     download_url = response["expiring_raw_log_url"]
-    if not download_url:
-        raise TaskException("Bitrise has no log to offer for job {}. Please check https://app.bitrise.io/app/{}".format(build_slug, BITRISE_APP_SLUG_ID))
```

Comment:
Because of this line, I ended up with a red job, even though Bitrise generated all locales. Their API just failed to provide us a link to their logs.